### ISSUE TITLE
Update hector_terrestrial_c.svg

### ISSUE THREAD

**Issue by bpbond** [JGCRI/hector#668]
In doing #667 I realized that the graphics file `vignettes/articles/hector_terrestrial_c.svg` is incomplete, as it doesn't include FFI or DACCS.

Do we have the source file for this figure? If not will need to recreate it.

<img width="803" alt="Screen Shot 2022-12-08 at 5 37 33 AM" src="https://user-images.githubusercontent.com/1956468/206426015-19d960c3-b14d-49dc-b5ff-5480e72cb443.png">

**Comment by bpbond** [comment]
@kdorheim Thoughts?

**Comment by kdorheim** [comment]
Should there be some sort of connection between NPP and LUC emissions?

**Comment by bpbond** [comment]
🤔  good idea

**Comment by bpbond** [comment]
Can this be closed?

**Comment by kdorheim** [comment]
I don't think so. Wouldn't we want the specific LUC veg/soil/detritus pool interactions to be added?
